common/quotas/global/algorithm/testdata/fuzz/FuzzMultiUpdate/c50ed3d1a22fe00d

```diff
@@ -0,0 +1,2 @@
+go test fuzz v1
```

Comment:
these "big" samples seemed small enough to keep, and they exercise things a bit harder than the trivially-small ones do.

but we could manually trim them and it'd be fine, if we feel strongly about keeping the size small, though before doing that I kinda want to make sure they didn't catch a NaN that the others missed.